content/concept-amendments.md

```diff
@@ -111,7 +113,8 @@ The following is a comprehensive list of all known amendments and their status o
 
 | Name                            | Introduced | Enabled |
 |---------------------------------|------------|---------|
-| [FlowV2](#flowv2)               | v0.32.1    | Expected 2016-08-24 |
+| [Flow](#flow)                   | TBD        | TBD |
+| [FlowV2](#flowv2)               | v0.32.1    | Vetoed |
```

Comment:
If my understanding is correct, "Vetoed" is always potentially just temporary. I know that's explained just above, but listing the status as "Vetoed" makes it seem slightly misleading. If I were just glancing at this, I'd assume (incorrectly) that the status was more permanent. Maybe link "Vetoed" to the section that explains how vetoing works (even though, yes, they're proximate)?